week4_approx/dqn_atari_pytorch.ipynb

```diff
@@ -1038,19 +1036,19 @@
    "source": [
     "## How to interpret plots:\n",
     "\n",
-    "This aint no supervised learning so don't expect anything to improve monotonously. \n",
-    "* **TD loss** is the MSE between agent's current Q-values and target Q-values. It may slowly increase or decrease, it's ok. The \"not ok\" behavior includes going NaN or stayng at exactly zero before agent has perfect performance.\n",
-    "* **grad norm** just shows the intensivity of training. Not ok is growing to values of about 100 (or maybe even 50) though it depends on network architecture.\n",
-    "* **mean reward** is the expected sum of r(s,a) agent gets over the full game session. It will oscillate, but on average it should get higher over time (after a few thousand iterations...). \n",
+    "This is not the supervised learning so don't expect anything to improve monotonously. \n",
+    "* **TD loss** is the MSE between agent's current Q-values and target Q-values. It may slowly increase or decrease, it's ok. The \"not ok\" behavior includes going NaN or staying at the exact zero before agent has perfect the performance.\n",
```

Comment:
This "the" is definitely redundant, although it could be moved to "before the agent".